README

```diff
@@ -109,8 +109,8 @@ Mandriva:
 	on that distro should be as easy as typing ``urpmi fio``.
 
 Arch Linux:
-        Arch Linux package is provided under community.
-        https://www.archlinux.org/packages/?sort=&q=fio&maintainer=&flagged=
+        Arch Linux package is provided under community sub-repository:
```

Comment:
I'd still recommend a minor change like "An Arch Linux package is provided under the Community sub-repository" to help guide the reader to your intent...